### ISSUE TITLE
python package local-attention release 1.9.15 causes ModuleNotFoundError: No module named 'torch.amp'

### ISSUE THREAD

**Issue by yitaochen** [thuml/Time-Series-Library#524]
Per the title says. Could you include local-attention==1.9.14 in the requirements.txt

**Comment by moured** [comment]
That was helpful!

**Comment by Musongwhk** [comment]
Hi. Thank you for your issue. 
Firstly, the module we use is **torch.cuda.amp**, not **torch.amp**; 
Secondly, local-attention is a dependency of other packages we require.If you configure the environment according to **requirements.txt**, the appropriate version should be automatically downloaded without the need for explicit settings.
If the version of your package local-attention causes ModuleNotFoundError, you may consider reconfiguring the appropriate version as you say. Please feel free to continue discussing if you have any further questions.

**Comment by yitaochen** [comment]
Yes, for some reason the author of local-attention packet made the following [change](https://github.com/lucidrains/local-attention/compare/1.9.14...1.9.15) between release 1.9.14 and 1.9.15:
```diff
- from torch.cuda.amp import autocast
+ from torch.amp import autocast
```
I was just thinking it'd be helpful if you can mention this somewhere in the doc, so people don't have to spend time on this issue.

**Comment by Musongwhk** [comment]
Thank you for your information! We also reproduce ModuleNotFoundError. If we configure the environment according to requirements.txt, local-attention version will be 1.9.15. 
Your discussion may be helpful to others who have similar doubts. We will consider mentioning this in requirements.txt or somewhere else.

**Comment by Et6an** [comment]
The same applies to CoLT5-attention, which also requires a lower version. I successfully ran it with version 0.10.20.

**Comment by HanZhaang** [comment]
> The same applies to CoLT5-attention, which also requires a lower version. I successfully ran it with version 0.10.20.

God you are my god
